core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -113,7 +115,7 @@ class DAGScheduler(
   private val failedEpoch = new HashMap[String, Long]
 
   private val dagSchedulerActorSupervisor =
-    env.actorSystem.actorOf(Props(new DAGSchedulerActorSupervisor(this)))
+    env.actorSystem.actorOf(Props(new DAGSchedulerActorSupervisor(this)), "DAGSchedulerSupervisor")
```

Comment:
I'm trying to push a commit on top of master, which I believe is later than what this pull request is on top of. On master, DAGSchedulerActorSupervisor is no longer a class. How will that affect this design?